contracts/multisig/README.md

```diff
@@ -11,6 +11,10 @@ As an example, let us imagine the following scenario. An institution launches a
 
 Thus, essentially the multisig SC (we will refer to it, from now on, as MSC) enables multiple parties to sign or approve an action that takes place - typically a requirement for certain wallets, accounts, and smart contracts to prevent a rogue or hacked individual from performing detrimental actions.
 
+## Developer's note
+
+Starting with the release version v0.45.5 of mx-contracts-rs, this contract has beed audited and changed to meet the required standards.
```

Comment:
I'd actually be more specific in saying that: release version v0.45.5 is the audited version for the multisig contract and link to it here: https://github.com/multiversx/mx-contracts-rs/releases/tag/v0.45.5

- this way there's no confusion weather other releases touch or change the code in some way - as we seen, releases that do not touch the multisig at all can break something in this mono-repo style